daprdocs/content/en/concepts/dapr-services/scheduler.md

```diff
@@ -5,17 +5,38 @@ linkTitle: "Scheduler"
+
+There is no concept of a leader Scheduler instance. All Scheduler service replicas are considered peers. All receive jobs to be scheduled for execution and the jobs are divvied up between the available Scheduler service replicas for trigger load balancing.
+
+The diagram below shows how the Scheduler service is used via the jobs API when called from your application. All the jobs that are tracked by the Scheduler service are stored in an embedded etcd database. 
 
 <img src="/images/scheduler/scheduler-architecture.png" alt="Diagram showing the Scheduler control plane service and the jobs API">
 
 ## Actor reminders
 
 Prior to Dapr v1.15, [actor reminders]({{< ref "actors-timers-reminders.md#actor-reminders" >}}) were run using the Placement service. Now, by default, the [`SchedulerReminders` feature flag]({{< ref "support-preview-features.md#current-preview-features" >}}) is set to `true`, and all new actor reminders you create are run using the Scheduler service to make them more scalable.
 
-When you deploy Dapr v1.15, any _existing_ actor reminders are migrated from the Placement service to the Scheduler service as a one time operation for each actor type. You can prevent this migration by setting the `SchedulerReminders` flag to `false` in application configuration file for the actor type.
+When you deploy Dapr v1.15, any _existing_ actor reminders are automatically migrated from the Placement service to the Scheduler service as a one time operation for each actor type. There will be _no_ loss of reminder triggers during the migration. However, you can prevent this migration and keep the existing actor reminders running using the Placement service by setting the `SchedulerReminders` flag to `false` in application configuration file for the actor type.
+
+## Job Triggering
+
+### Job Ordering
+
+When the Scheduler service triggers a job there is no guarantee of job trigger ordering, meaning we do not guarantee FIFO or LIFO trigger ordering. 
+
+### Job Failure Policy and Staging Queue
+
+When the Scheduler service triggers a job and it has a client side error, with the failure policy, the job is retried by default with a 1s interval and 3 maximum retries. A failure policy can be configured for a consistent retry or to drop a job. 
+- Actor reminder type jobs retry forever until successful completion.
+- Workflow reminders are one time, meaning they only trigger once successfully with a 1 second retry interval.
+
+For non-client side errors, for example, when a job cannot be sent to an available Dapr sidecar at trigger time, it is placed in a staging queue within the Scheduler service. Jobs remain in this queue until a suitable sidecar instance becomes available, at which point they are automatically sent to the appropriate Dapr sidecar instance. Jobs are sent back to a single replica for the same app ID that scheduled the job in a round robin manner.
```

Comment:
We should not mention round robin anywhere. The docs should only read "randomly load balanced".

Yes, technically there is a round robin implementation, but the logic is (kinda intentionally) incredibly fragile and will be reset during any connection event. Users _should not_ rely or try to coerce job scheduling to become round robin, and instead assume completely random.

For users who wish to have locality for their jobs, they should instead use actor reminders.

For users who wish to have jobs triggered on the same host which created it, they should create an actor reminder using an actor type that is a random UUID unique to that replica to enforce perfect locality.